core/src/migrations/000083-models.ts

```diff
@@ -1,20 +1,40 @@
 import Sequelize from "sequelize";
 
-const default_model = "mod_profiles";
+const default_model = {
+  id: "mod_profiles",
+  name: "Profiles",
+  type: "profile",
+};
 
 export default {
   up: async (
     queryInterface: Sequelize.QueryInterface,
     DataTypes: typeof Sequelize
   ) => {
+    let toCreteModel = false;
+    const [sources] = await queryInterface.sequelize.query(
+      `SELECT * FROM "sources" WHERE "locked" IS NOT NULL`
+    );
+    const [destinations] = await queryInterface.sequelize.query(
+      `SELECT * FROM "destinations" WHERE "locked" IS NOT NULL`
+    );
```

Comment:
There a reason these are raw sql over sequelize?

Reply:
Yeah! Because in the migrations the models aren't loaded/ready yet. We could make "stub" models against the table in question, but I've usually seen that backfire because the expected associations/validations won't be there because it's not a "real" model instance